### PR TITLE
Update dependencies and regenerate package-lock.json

### PR DESCRIPTION
**Pull request by XhmikosR**
* debug       ^4.1.1  →   ^4.2.0
* got        ^11.6.0  →  ^11.7.0
* penthouse   ^2.3.1  →   ^2.3.2
* postcss    ^7.0.32  →   ^7.0.35
* prettier    ^2.1.1  →   ^2.1.2
* vinyl       ^2.2.0  →   ^2.2.1
* jest       ^26.4.2  →  ^26.5.0

Only 2 deps are left: postcss and tempy; feel free to update those too @bezoerb :)

### CODE REVIEW

**Review by bezoerb — APPROVED** [review]
LGTM

**Comment by bezoerb** [comment]
Updated the other deps and changed the supported node versions according to postcss.

File: package.json [hunk 1]
```diff
@@ -23,19 +23,19 @@
     "optimization"
   ],
   "engines": {
-    "node": ">=10"
+    "node": "^10 || ^12 || >=14"
```
Comment (bezoerb) [inline package.json:26]: @XhmikosR: Do you think we need a major version bump for this one?

Comment (XhmikosR) [reply package.json:26]: Technically, yeah. Even though I don't think people should use non-LTS releases, this has happened before with autoprefixer or a dep it was depending on. Hence why I didn't touch postcss in this PR :)

**Pull request merged**